admin/react/pages/__tests__/ApiKeysPage.test.jsx

```
@@ -0,0 +1,248 @@
+            <div key={column.key || column.label || index}>
+              {column.render ? column.render(row) : row[column.key]}
+            </div>
+          ))}
+        </div>
+      ))}
+    </div>
+  ),
+  LoadingTable: () => <div>Loading API keys</div>,
+}));
+
+jest.mock("../../components/MutationDialog.jsx", () => ({
```

ESLint

[error] 'jest' is not defined.

(no-undef)